lib/views/pages/profile/posts_info/popup/post_popup.dart

```diff
@@ -0,0 +1,48 @@
+import "package:flutter/material.dart";
+
+class PostPopUp extends StatelessWidget {
+  static const postPopUpTitleKey = Key("postPopUpTitle");
+  static const postPopUpDescriptionKey = Key("postPopUpDescription");
+  static const postPopUpDeleteButtonKey = Key("postPopUpDeleteButton");
+
+  const PostPopUp({
+    super.key,
+    required this.title,
+    required this.description,
+  });
+
+  final String title;
+  final String description;
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: Text(
+        key: postPopUpTitleKey,
+        title,
+      ),
```

Comment:
Very nice job with the PopUp !
I just think that we could force the post title to be on a single scrollable line to avoid an overflow in the case of a very long post title.

```suggestion
      title: SingleChildScrollView(
        scrollDirection: Axis.horizontal,
        child: Text(
          key: postPopUpTitleKey,
          maxLines: 1,
          title,
        ),
      ),
```

Reply:
Good idea !